packages/mds-compliance-engine/batch_process.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2021 City of Los Angeles
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import cache from '@mds-core/mds-agency-cache'
+import { ComplianceServiceClient } from '@mds-core/mds-compliance-service'
+import db from '@mds-core/mds-db'
+import logger from '@mds-core/mds-logger'
+import { ModalityPolicy, ModalityPolicyTypeInfo } from '@mds-core/mds-types'
+import { processPolicy } from './engine'
+import { getSupersedingPolicies } from './engine/helpers'
+
+async function main() {
+  await cache.startup()
+  const policies: ModalityPolicy[] = getSupersedingPolicies(await db.readActivePolicies<ModalityPolicyTypeInfo>())
+  const geographies = await db.readGeographies({ get_published: true })
+  const results = await Promise.all(
+    policies.map(policy => {
+      return processPolicy(policy, geographies)
+    })
+  )
```

Comment:
I don't think that this will run well in a prod environment -- processing all policies in parallel in one process (each call resulting in fetching a ton of devices/events/telem and storing in memory) seems like it'll cause an OOM issue, or at the very least result in ultra slow compute. If we really wanna do this in one pod, I'd recommend looking into some concurrency libraries such as https://www.npmjs.com/package/p-map